uw_gws/__init__.py

```diff
@@ -98,7 +98,7 @@ def create_group(self, group):
         """
         self._valid_group_id(group.id)
 
-        body = {"data": group.json_data()}
+        body = {"data": get_group_json(group)}
```

Comment:
I would replace this with {k: v for k, v in group.json_data.items() if v is not None}